src/index.html

```diff
@@ -1,17 +1,174 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8">
+    <meta charset="UTF-8" />
     <meta
       name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <title>HTML Form</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./style.css" />
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+      <fieldset>
+        <legend>Personal information</legend>
+        <div class="field">
+          <label>
+            Surname:
+            <input
+              type="text"
+              name="Surname"
+              autocomplete="off"
+              required
+            >
+          </label>
+        </div>
+        <div class="field">
+          <label>
+            Name:
+            <input
+              type="text"
+              name="Name"
+              autocomplete="off"
+              required
+            >
+          </label>
+        </div>
+        <div class="field">
+          <label>
+            How old are You?
+            <input
+              type="number"
+              name="number"
+              min="1"
+              max="100"
+              value="12"
+            >
+          </label>
+        </div>
+        <div class="field">
+          <label>
+            Full date of birth:
+            <input type="date" name="date" />
+          </label>
+        </div>
+        <div class="field">
+          <label>
+            I accept the term of the agreement
+            <input
+              type="checkbox"
+              name="acceptTerm"
+              required
+            >
+          </label>
+        </div>
+      </fieldset>
+      <fieldset>
+        <legend>Registration</legend>
+        <div class="field">
+          <label>
+            E-mail:
+            <input
+              type="email"
+              name="email"
+              placeholder="email@example.com"
+            >
+          </label>
+        </div>
+        <div class="field">
+          <label>
+            Password:
+            <input type="password" name="password" />
+          </label>
+        </div>
+      </fieldset>
+      <fieldset>
+        <legend>An interesting fact about you!</legend>
+        <div class="field">
+          Do you love cats?
+          <input
+            type="radio"
+            id="button_love_cats"
+            name="Cats"
+            value="Yes"
+          >
+          <label for="button_love_cats">Yes</label>
```

Comment:
IMHO, it's more pretty))

```suggestion
          Do you love cats?
          <label>
	          <input
	            type="radio"
	            name="Cats"
	            value="Yes"
	          >
	          Yes
          </label>
```